uportal-war/src/main/java/org/jasig/portal/rest/PeopleRESTControllerV2.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.jasig.portal.rest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.jasig.portal.portlets.lookup.PersonLookupHelperImpl;
+import org.jasig.portal.security.IPerson;
+import org.jasig.portal.security.IPersonManager;
+import org.jasig.services.persondir.IPersonAttributes;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Controller
+@RequestMapping("/people/v2")
```

Comment:
Making the APIs easier to consume is a good thing.  +1 on merging and for your insight.  I have three minor requests first though:
* change the url to /v4-3/people to be consistent with how we are naming the endpoints (see https://github.com/Jasig/uPortal/blob/master/uportal-war/src/main/java/org/jasig/portal/rest/layout/LayoutJsonV43RenderingController.java#L70 as an example)
* Add a @since 4.3 to this class and to the added method above to clarify their entry into the API
* Add a bit of javadoc to the methods describing the output, perhaps a sample output (our APIs are sorely lacking javadocs explaining what the output looks like).